packages/test/test-service-load/testConfig.json

```diff
@@ -12,16 +12,16 @@
             "faultInjectionMaxMs": 150000,
             "optionOverrides":{
                 "odsp":{
-                    "loader":{
-                        "summarizeProtocolTree": [false]
+                    "configurations":{
+                        "Fluid.Driver.Odsp.EnableContainerTypeSummaryUpload": [true],
+                        "Fluid.Container.summarizeProtocolTree": [true]
```

Comment:
the value here should be [true,false]. we need to ensure that everything works when either mode, as when we rollout there will be clients uploading both kinds until it saturates.